source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -248,5 +250,93 @@ bool SetSpecConstantDefaultValuePass::Process(ir::Module* module) {
   return modified;
 }
 
+// Returns true if the char pointed by the given pointer is '\0' or '}'.
+bool IsEnd(const char* const ptr) {
```

Comment:
guard against `ptr == nullptr`? Or should be clear in the comment `ptr` should be non-`nullptr`.

Besides, this can be labelled as `inline` function.

`return *ptr == '\0' || *ptr == '}'`